components/src/core/components/Comments/Comment.vue

```diff
@@ -420,7 +408,7 @@ export default defineComponent({
     const confirmDeleteButtonData = computed(() => {
       const initialObject = {
         label: 'Yes, Delete',
-        iconName: 'oxd-trash',
+        iconName: null,
```

Comment:
No need to pass this param @MifdhaM right?

Reply:
Yes. removed